packages/vuetify/src/util/__tests__/mergeData.spec.ts

```diff
@@ -0,0 +1,69 @@
+import { mergeClasses, mergeListeners, mergeStyles } from '../mergeData'
```

Comment:
Is this a candidate for https://jestjs.io/docs/en/api#testeachtablename-fn-timeout?

Reply:
and by this I mean the entire file.

Reply:
Hmm, sounds interesting. This can help cleaning up the file and simplifying adding new tests to it.